rfc9849.md

```diff
@@ -1331,16 +1435,46 @@ when different endpoints might have different configurations.
 
 ### Middleboxes
 
+<!--[rfced] How may we update this sentence to make it clear whether
+all the requirements or only some of the requirements require 
+proxies to act as conforming TLS client and server?
+
+For background, in general, the RPC recommends using nonrestrictive "which" 
+and restrictive "that". (More details are on 
+https://www.rfc-editor.org/styleguide/tips/) However, edits to that 
+usage have not been made in this document. In this specific sentence, 
+we are asking about the usage because it can affect the understanding 
+of the statement.
+
+Original:
+  The requirements in [RFC8446], Section 9.3 which require proxies to
+  act as conforming TLS client and server provide interoperability with
+  TLS-terminating proxies even in cases where the server supports ECH
+  but the proxy does not, as detailed below.
+
+Option A (all requirements require it):
+  The requirements in [RFC8446], Section 9.3, which require proxies to
+  act as conforming TLS client and server, provide interoperability with
+  TLS-terminating proxies even in cases where the server supports ECH
+  but the proxy does not, as detailed below.
+
+Option B (some requirements require it):
+  The requirements in [RFC8446], Section 9.3 that require proxies to
+  act as conforming TLS client and server provide interoperability with
+  TLS-terminating proxies even in cases where the server supports ECH
+  but the proxy does not, as detailed below.
+-->
+
```

Comment:
I don't think interpretation of this text should depend on "which" vs. "that". Rather the text should be clear.

I'm not sure that this distinction actually matters, but under the assumption that it does, the first and second bullets seem not to apply to proxies at all.

```
  [RFC8446], Section 9.3 requires proxies to act as conforming TLS client and
  servers. This provides interoperability with TLS-terminating proxies even in
  cases where the server supports ECH but the proxy does not, as detailed below.
```